kbs/src/api/src/attestation/coco/grpc.rs

```diff
@@ -125,6 +126,27 @@ impl Attest for GrpcClientPool {
 
         Ok(token)
     }
+
+    async fn generate_supplemental_challenge(
+        &self,
+        tee: Tee,
+        tee_parameters: Option<Vec<u8>>,
+    ) -> Result<String> {
+        let req = tonic::Request::new(ChallengeRequest {
+            tee: to_grpc_tee(tee).into(),
+            tee_params: String::from_utf8(tee_parameters.unwrap()).unwrap(),
```

Comment:
We'd try to avoid using `unwrap()`s. See https://github.com/confidential-containers/trustee/blob/main/attestation-service/attestation-service/src/bin/grpc/mod.rs#L194-L195 for example

Reply:
Fixed